DDDEastAnglia/Controllers/SponsorModelQuery.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using DDDEastAnglia.DataAccess;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+using DDDEastAnglia.Models;
+
+namespace DDDEastAnglia.Controllers
+{
+    public class SponsorModelQuery
+    {
+        private readonly ISponsorRepository sponsorRepository;
+
+        public SponsorModelQuery(ISponsorRepository sponsorRepository)
```

Comment:
Not in production, I don't think.  I have a branch in my fork called something like  `repositories-to-query-objects` that might provide some inspiration...